elastic-utils-lib/project.clj

```diff
@@ -8,7 +8,7 @@
                [potemkin]]
   :dependencies [[cheshire "5.8.1"]
                  [clj-http "2.3.0"]
-                 [clojurewerkz/elastisch "2.2.2"]
+                 [clojurewerkz/elastisch "5.0.0-beta1"]
```

Comment:
I'm a little wary of using an unstable library that has a history of extremely slow updates for something this important.  We are potentially trading reliability for expedience, but I wasn't a huge fan of using spandex either.  If this works then great, it just makes me nervous.

Reply:
We always have the option to roll our own. This makes it more convenient now that we have ruled out Spandex.